projects/sport-data-valley/seeds/protocols/psychological_competencies.rb

```diff
@@ -0,0 +1,36 @@
+default_protocol_duration = 1.day  # evt eerder dynamisch afbreken
+
+pr_name = 'psychological_competencies'
+protocol = Protocol.find_by(name: pr_name)
+protocol ||= Protocol.new(name: pr_name)
+
+protocol.duration = default_protocol_duration
+protocol.informed_consent_questionnaire = nil
+protocol.invitation_text = 'Je bent uitgenodigd door je coach om een vragenlijst in te vullen.'
```

Comment:
do we need to translate this? how do we do it on the other questionnaires?

Reply:
Good question. Let me look this up.

Reply:
Okay, so this is just the default invitation text if none is provided. However, it can be overridden when starting a protocol from base platform (you then specify both an English and a Dutch invitation text).

Reply:
I'll add a comment.

Reply:
Fixed.